modules/meeting/spec/features/recurring_meetings/recurring_meeting_crud_spec.rb

```diff
@@ -89,13 +89,17 @@
   it "can delete a recurring meeting from the show page and return to the index page" do
     show_page.visit!
 
-    click_on "recurring-meeting-action-menu"
+    show_page.delete_meeting_series
+    show_page.in_delete_dialog do
+      page.check "I understand that this deletion cannot be reversed"
 
-    accept_confirm(I18n.t("text_are_you_sure")) do
-      click_on "Delete meeting series"
+      click_on "Delete permanently"
     end
 
     expect(page).to have_current_path meetings_path # check path
```

Comment:
Hi @oliverguenther. I saw you added this comment. Is that a `TODO` (i.e. do we still need to check if this is the correct path), or just an explanation of what the line does? - it seems unclear to me if we always want to forward to the global meetings path after meeting deletion

Reply:
`RecurringMeetinsController#destroy` redirects to `polymorphic_path([@project, :meetings])`, so either the assertion or the implementation must be wrong? 😕

Reply:
We want to keep the current project context, if you are in one.

So if you are looking at `/projects/my-identifier/meetings/1234`, and start deleting from there, I would expect you to be redirected to `/projects/my-identifier/meetings`.

Likewise when destroying the meeting from the global meeting page `/meetings/1234`, you should be redirected to `/meetings`. It could be that we're missing the correct implementation here, or that the spec is incorrect. I will double check the current behavior